packages/class-core-build/artifacts/TestLanguage.txt

```diff
@@ -0,0 +1,42 @@
+language TestLanguage
+    version: 0
+    entities (↓name):
```

Comment:
Does the arrow down character has a specific meaning or is it just to test that non common characters are supported?

Reply:
It implicitly tests whether Unicode is properly supported, but its meaning is that the entities are sorted by the entities’ names. This ensures stability of the textualization in the face of other structural changes to the language(s), like reordering features or effectively reordering features by changing the inheritance.

Reply:
Nice!